<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Customer</class>
 <widget class="QDialog" name="Customer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Passenger </string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLineEdit {
    padding: 10px;
    border: 2px solid #0096C9;
    border-radius: 5px;
    font-size: 16px;
    font-family: Arial;
}




</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>70</y>
     <width>501</width>
     <height>331</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>441</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Username</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_customer_username"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Password</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_customer_password">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_customer_login">
       <property name="styleSheet">
        <string notr="true">QPushButton {
  background-color: #1E90FF;
  color: white;
  border-radius: 5px;
  padding: 10px;
  font-weight: bold;
  border: 2px solid black;
}

QPushButton:hover {
  background-color: rgb(166, 98, 200);
  font-style: italic;
  border: 2px solid white;
  color:black;
}

QPushButton:pressed {
  background-color: #008080;
  font-weight: bold;
  color: white;
  border: 2px solid white;
}
</string>
       </property>
       <property name="text">
        <string>Log In</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Passenger Login</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_customer_back">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>440</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
  background-color: rgb(255, 134, 162);
  color: white;
  border-radius: 5px;
  padding: 5px;
  font-weight: bold;
  border: 2px solid black;
}

QPushButton:hover {
  background-color: rgb(166, 98, 200);
  font-style: italic;
  border: 2px solid white;
 color:black;
}

QPushButton:pressed {
  background-color: #008080;
  font-weight: bold;
  color: white;
  border: 2px solid white;
}
</string>
   </property>
   <property name="text">
    <string>BACK</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_customer">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>461</width>
     <height>331</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
